src/routes/gallery/find.js

```diff
@@ -0,0 +1,44 @@
+import Bluebird from 'bluebird'
+
+import models from '../../models'
+
+export default {
+
+  method: 'get',
+  path: '/galler(y|ies)',
+
+  handler(request, response) {
+
+    return models.Gallery.find()
```

Comment:
In what situation is this route used?

Reply:
The "gallery list view" that lists all the galleries. Currently it'll only return the single Codex Gallery, but this is so people can eventually browse all available galleries.